src/main/java/com/minecolonies/coremod/items/ModItems.java

```diff
@@ -28,10 +30,37 @@
     /**
      * Private constructor to hide the implicit public one.
      */
-    private ModItems()
+    public ModItems()
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Hide this public constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1118)